### PR TITLE
Dfpl 1935 closure dates not backdated redeploy

### PR DESCRIPTION
**Pull request by dawudgovuk**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-1935

### Change description ###
INC5568344 Closure dates not being backdated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by qasimnawaz-moj — DISMISSED** [review]

**Review by jamesrferguson1 — APPROVED** [review]

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGeneratorTest.java [hunk 1]
```diff
@@ -120,16 +125,50 @@ private CaseData buildCaseData(String closeCase, String isFinalOrder, Order orde
     @ParameterizedTest
     @MethodSource("generateOrderStates")
     void shouldCloseCaseWhenOrderIsDefaultFinal(String isFinalOrder) {
-        CaseData caseData = buildCaseData("Yes", isFinalOrder, C32A_CARE_ORDER);
+        CaseData caseData = buildCaseData("Yes", isFinalOrder, C32A_CARE_ORDER, APPROVAL_DATE, null);
```
Comment (GauravChawlaMOJ) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGeneratorTest.java:129]: Just minor thing: can we add the test when both approval_date and approval_date_time are not null?

Comment (dawudgovuk) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/ManageOrdersClosedCaseFieldGeneratorTest.java:129]: Cool. Added a test. Could you approve?

**Review by GauravChawlaMOJ — APPROVED** [review]
